### PR TITLE
Create README for test to validate fan tray type.

### PR DESCRIPTION
**Pull request by SydneyCaulfeild**

### CODE REVIEW

File: feature/gnmi/otg_tests/telemetry_fan_tray_type_test/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# gNMI-??: Telemetry: Fan Tray Type Test
```
Comment (SydneyCaulfeild) [inline feature/gnmi/otg_tests/telemetry_fan_tray_type_test/README.md]: @dplore do you know where I find what gNMI version to use here?

File: feature/gnmi/otg_tests/telemetry_fan_tray_type_test/README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  ## State Paths ##
+  /components/component/state/type:
+
+rpcs:
+  gnmi:
+    gNMI.Subscribe:
+```
+
+
+## Minimum DUT platform requirement
```
Comment (SydneyCaulfeild) [inline feature/gnmi/otg_tests/telemetry_fan_tray_type_test/README.md:30]: @dplore I am not familiar with DUT platform requirements, but it looks like some other tests leave this as N/A.

File: feature/gnmi/otg_tests/telemetry_fan_tray_type_test/README.md (hunk 1 above)
Comment (dplore) [reply feature/gnmi/otg_tests/telemetry_fan_tray_type_test/README.md]: You can select an unused number from [testregistry.textproto](https://github.com/openconfig/featureprofiles/blob/main/testregistry.textproto)

File: feature/gnmi/otg_tests/telemetry_fan_tray_type_test/README.md (hunk 2 above)
Comment (dplore) [reply feature/gnmi/otg_tests/telemetry_fan_tray_type_test/README.md:30]: For this, "MFF" should be used.  (Modular form factor devices). This is expected to be a chassis with removable line cards, fabrics, controller_card and in this case, FAN_TRAY.  Of course, this is not the only type of device that might have a FAN_TRAY, but is the most common which would.   This documentation is a hint to the test runner which class of hardware should be used.

Comment (SydneyCaulfeild) [reply feature/gnmi/otg_tests/telemetry_fan_tray_type_test/README.md:30]: Ok thank you!

**Comment by dplore** [comment]
You can merge this here, which will update your personal copy of the featureprofiles repo.  
You will still need to create a pull request at https://github.com/openconfig/featureprofiles

**Pull request merged**